elements/classpower.lua

```diff
@@ -113,10 +113,17 @@ local function Update(self, event, unit, powerType)
 			mod = UnitPowerDisplayMod(ClassPowerID)
 		end
 
+		local numActive = math.ceil(mod == 0 and 0 or cur / mod)
```

Comment:
Nvm, I just saw the performance difference, math.ceil is 7-10 times slower than that addition!